rustls/src/sign.rs

```diff
@@ -252,7 +252,7 @@ impl Signer for RsaSigner {
 /// A SigningKey that uses exactly one TLS-level SignatureScheme
 /// and one ring-level signature::SigningAlgorithm.
 ///
-/// Compare this to RSASigningKey, which for a particular key is
+/// Compare this to RsaSigningKey, which for a particular key is
```

Comment:
Maybe [`RsaSigningKey`]?

Reply:
I'd rather just keep this PR scoped to fixing the typos. If we want to add intra-crate links to the docs, which I think is a fine idea, we should do that more thoroughly in another PR.